packages/manager/apps/key-management-service/src/components/credential/credentialDatagrid/CredentialDatagridCells.tsx

```diff
@@ -12,12 +12,14 @@ import { CredentialStatus } from '../credentialStatus/CredentialStatus.component
 export const DatagridCredentialCellName = (credential: OkmsCredential) => {
   const navigate = useNavigate();
   return (
-    <Links
-      onClickReturn={() => {
-        navigate(`${credential.id}`);
-      }}
-      label={credential.name}
-    />
+    <div>
```

Comment:
Returning Links without a div wrapper is having a side effect ?

Reply:
yes, the tab froze when we click on the link without the div wrapper.